src/components/EntityPage/Participant/ParticipantSummary.js

```diff
@@ -157,6 +157,16 @@ function specimenSummaryTableData(specimen) {
 
 const ParticipantSummary = ({ participant }) => {
   const specimens = get(participant, 'biospecimens.hits.edges', []);
+  const hasFile = get(participant, 'files.hits.edges', [] ).length === 0 ? false : true
+  let wrongTypes = ['Aligned Reads', 'gVCF', 'Unaligned Reads', 'Variant Calls'];
+
+  let hasSequencingData = false
+  for(const i in get(participant, 'files.hits.edges', [] )){
```

Comment:
Could we use `let` in this case?

Reply:
for which variable ?

Reply:
A more simple way to do this could be:
`https://lodash.com/docs/4.17.15#some`
Try it out sometimes. :)

Reply:
For `for(const i`